src/style.css

```diff
@@ -132,3 +132,26 @@ button {
   font-size: 1.7em;
   text-align: center;
 }
+
+.meal-popup {
+  position: fixed;
+  top: 10px;
+  left: 10px;
+  background-color: white;
+  border: 4px solid #3e3e3e;
+  overflow-y: scroll;
+  height: 100%;
+}
```

Comment:
[Optionaly] I noticed that when the popup window appears and I scroll down the footer appears on the front of the popup, so I suggest you use the z-index property and assign it to 999 for example to make the pop appears in the front of other parts.👌👍